libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -41,9 +41,27 @@ export class FunctionClient implements IFunctionClient {
   }
 
   /**
-   * Invoke the function, passing the given payload as an argument.
-   *
-   * @returns the function returned payload and logs
+   * Invokes the function asynchronously, passing the given payload as an argument.
+   * @returns immediately once the event has been handed off to AWS Lambda.
+   */
+  public async invokeAsync(payload: string): Promise<void> {
+    const command = new InvokeCommand({
+      FunctionName: this.functionArn,
+      Payload: fromUtf8(JSON.stringify(payload)),
+      InvocationType: "Event",
+    });
+    const response = await this.lambdaClient.send(command);
+
+    if (response.StatusCode !== 202) {
+      throw new Error(
+        `Failed to enqueue event. Received status code: ${response.StatusCode}`
+      );
+    }
```

Comment:
Maybe we can log any extra error information available in case it fails. I found these fields listed in the docs [here](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-lambda/Interface/InvokeCommandOutput/)
```suggestion
    if (response.StatusCode !== 202) {
      console.error("Error: " + response.FunctionError);
      console.error(fromUtf8(response.Payload));
      throw new Error(
        `Failed to enqueue event. Received status code: ${response.StatusCode}`
      );
    }
```